src/ffi/ios/structs.rs

```diff
@@ -0,0 +1,177 @@
+use std::{
+    borrow::Cow,
+    collections::HashMap,
+    convert::TryFrom,
+    mem::{self, ManuallyDrop},
+};
+
+use anyhow::Context;
+use core_foundation::{
+    array::CFArray,
+    base::{FromVoid, TCFType},
+    data::{CFData, CFDataRef},
+    dictionary::{CFDictionary, CFDictionaryRef},
+    string::{CFString, CFStringRef},
+    url::{CFURLRef, CFURL},
+};
+use http::{
+    header::{HeaderName, HeaderValue},
+    Uri,
+};
+
+#[repr(C)]
+pub enum Method {
+    // TODO more methods!
```

Comment:
I thought that @cgrigis successfully also uses `POST` - should this be added here?

Reply:
Ho right, adding the other methods.